src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -1115,12 +1151,14 @@ export default class DataTable extends Vue {
     const classes = this._dataTableClasses(this.config.style, this._sortable),
       head = this._head(),
       toolbar = this._toolbar(),
+      bulkaction = this._bulkAction(),
       body = this._body(),
       pagination = this._pagination();
 
     return (
       <div class={classes} role="table" ref="dataTable" data-table-number={dataTableNumber}>
         {toolbar}
+        {bulkaction}
```

Comment:
How do we handle this?

https://user-images.githubusercontent.com/25136131/146739093-ab69e5ba-36f5-4f42-a344-1df1f6c8ec37.mov